vm-virtio/src/queue.rs

```diff
@@ -218,6 +218,9 @@ pub struct Queue {
     /// Inidcates if the queue is finished with configuration
     pub ready: bool,
 
+    /// MSI-X vector of the queue
+    pub msix_vector: u16,
```

Comment:
So you are adding some PCI concepts to virtio core. This breaks your assumption that virtio code other than virtio-pci should not aware the existence of PCI. Suggest to change this to 'queue_vector' so even in future we may add queue vector support for mmio transport.

Reply:
Yeah I know, but the queue needs to contain this information, even if it's ignored in the context of virtio-mmio. I'm okay renaming this. I'd say `vector` is enough since that's already part of the Queue structure.